### ISSUE TITLE
As a user, I have a user profile where I describe myself in a few sentences and can also add a link to my GitHub profile. 

### ISSUE THREAD

**Issue by AlexHartveld** [RefugeesCodeAT/hackboard#53]
I can also add certain tags to my profile, the same ones that are used in the project overview, see #50. 
With Angular Components this can be implemented.

**Comment by rainerhahnekamp** [comment]
Nope, this issue is about the user itself. Not the project. Add following properties to the User entity and table.

- [ ] GitHub
- [ ] Description
- [ ] Tags (using a `@ManyToMany`)
- [ ] This fields can be entered in the user's account page.
- [ ] There must a be user profile page where others - like the project owner - can see this information.
